### PR TITLE
Add random, fixed and question-only agents to the benchmark

### PR DESCRIPTION
**Pull request by kongzii**

### CODE REVIEW

**Comment by kongzii** [comment]
[benchmark_report_m50_manifold.pdf](https://github.com/polywrap/evo.researcher/files/14236261/benchmark_report_m50_manifold.pdf)

File: evo_researcher/functions/research.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def research(
     openai_key: str,
     use_summaries: bool,
     model: str = "gpt-4-1106-preview",
-    initial_subqueries_limit: int = 20,
+    initial_subqueries_limit: int = 4,
```
Comment (evangriffiths) [inline evo_researcher/functions/research.py:16]: Is this meant to be part of this PR? 👀

Comment (kongzii) [reply evo_researcher/functions/research.py:16]: No, it's not! Sorry, reverted.

Comment (kongzii) [reply evo_researcher/functions/research.py]: Good catch, fixed in all chained PRs.

**Review by evangriffiths — APPROVED** [review]
LGTM!

**Pull request merged**